python3-gceimgutils.spec

```diff
@@ -26,16 +26,32 @@ License:        GPL-3.0+
 Group:          System/Management
 Url:            https://github.com/SUSE-Enceladus/gceimgutils
 Source0:        %{upstream_name}-%{version}.tar.bz2
+%if 0%{?sle_version} >= 150400
```

Comment:
The file should be renamed to `python-gceimgutils` and let's setup the spec file like ec2imgutils https://github.com/SUSE-Enceladus/ec2imgutils/blob/master/python-ec2imgutils.spec

Reply:
Updated. Should be ready when 3.11 switchover happens.

Reply:
Wasn't sure if the libalternatives are required here.

Reply:
Spec was merged separately. Rebased this PR to pull in spec.